site/src/component/Review/Review.tsx

```diff
@@ -57,6 +59,20 @@ const Review: FC<ReviewProps> = (props) => {
     sortedReviews = reviewData.slice(0);
   }
 
+  // calculate frequencies of professors or courses in list of reviews
```

Comment:
Note for other reviewers: This reordering fixes the issue where when filtering by a certain prof/course for reviews, other profs/courses would always show that they had zero reviews, even when they didn't.